Library/Homebrew/formula.rb

```diff
@@ -1455,16 +1455,6 @@ def system(cmd, *args)
       $stdout.flush
 
       unless $?.success?
-        log_lines = ENV["HOMEBREW_FAIL_LOG_LINES"]
-        log_lines ||= "15"
-
-        log.flush
-        if !verbose || verbose_using_dots
-          puts "Last #{log_lines} lines from #{logfn}:"
-          Kernel.system "/usr/bin/tail", "-n", log_lines, logfn
-        end
-        log.puts
-
```

Comment:
Agreed. Also, I do found this function useful to me. So how about this:

* Keeping the original function if HOMEBREW_FAIL_LOG_LINES is set
* Otherwise, only show build is failing.